public/controllers/register-agent/containers/steps/steps.tsx

```diff
@@ -0,0 +1,244 @@
+            >
+              {popoverServerAddress}
+            </EuiPopover>
+          </EuiFlexItem>
+        </EuiFlexGroup>
+      ),
+      children: (
+        <Fragment>
+          <EuiFlexGroup gutterSize='s' wrap>
+            {REGISTER_AGENT_DATA_STEP_TWO.map((data, index) => (
+              <EuiFlexItem key={index}>
+                <EuiText className='stepSubtitleServerAddress'>
+                  {data.subtitle}
+                </EuiText>
+              </EuiFlexItem>
+            ))}
+          </EuiFlexGroup>
+          <InputForm {...form.fields.serverAddress} label={<></>} />
+        </Fragment>
+      ),
+      status: !form.fields.operatingSystemSelection.value
+        ? 'disabled'
+        : !form.fields.serverAddress.value &&
+          form.fields.operatingSystemSelection.value
+        ? 'current'
+        : form.fields.operatingSystemSelection.value &&
+          form.fields.serverAddress.value
+        ? 'complete'
+        : '',
+    },
+    ...(!(!needsPassword || hidePasswordInput)
+      ? [
+          {
+            title: (
+              <EuiTitle className='stepTitle'>
+                <p>Wazuh password</p>
+              </EuiTitle>
+            ),
+            children: <Fragment>{'Phrase to define'}</Fragment>,
+          },
+        ]
+      : []),
+    {
+      title: (
+        <EuiTitle className='stepTitle'>
+          <p>Optional settings</p>
+        </EuiTitle>
+      ),
+      children: (
+        <Fragment>
+          <EuiFlexGroup gutterSize='s' wrap>
+            {REGISTER_AGENT_DATA_STEP_THREE.map((data, index) => (
+              <EuiFlexItem key={index}>
+                <EuiText className='stepSubtitle'>{data.subtitle}</EuiText>
+              </EuiFlexItem>
+            ))}
+          </EuiFlexGroup>
+          <InputForm
+            {...form.fields.agentName}
+            label={
+              <>
+                <EuiFlexGroup>
+                  {/* <EuiFlexItem
+                    style={{
+                      flexDirection: 'row',
+                      fontStyle: 'normal',
+                      fontWeight: 700,
+                      fontSize: '12px',
+                      lineHeight: '20px',
+                      color: '#343741',
+                    }}
+                  >
+                    Assign an agent name{' '}
+                    <EuiIconTip
+                      className='iconTooltip'
+                      content='Source maps allow browser dev tools to map minified code to the original source code'
+                      position='right'
+                    />
+                  </EuiFlexItem> */}
+                  <EuiFlexItem grow={false}>
+                    <EuiPopover
+                      button={
+                        <EuiButtonEmpty
+                          iconType='questionInCircle'
+                          iconSide='right'
+                          onClick={onButtonAgentName}
+                          // className='subtitleAgentName'
```

Comment:
**nitpick**: remove it

Reply:
Removed in commit [777d15d](https://github.com/wazuh/wazuh-kibana-app/pull/5554/commits/777d15df38e5e1f1d42b05b49a9cdd8f3ccbf29d)